README.md

```diff
@@ -257,7 +257,7 @@ The system is tested on Ubuntu, but should probably work on other operating syst
 ## Setup
 
 * Install the runtime dependencies
-  * `pip install -r requirements.txt`
+  * `pipenv sync --system`
```

Comment:
```suggestion
  * `pipenv sync`
  * `pipenv shell`
```

Might as well make use of a venv now we can easily

Reply:
Make sense. Just curious why are not using `pipenv sync` and `pipenv shell` in GHA | Dockerfile, but `pipenv sync --system`

Reply:
Because in that case, we do want to install files to the system python rather than a virtualenv. There's only one project in the Dockerfile so there's no need to worry about dependency collisions

Reply:
:+1: